luda/editor/client/src/timeline/type.ts

```diff
@@ -11,14 +11,20 @@ export type TimelineState = {
     msPerPixel: number;
     timeRulerHeight: number;
   };
-  tracks: Track[];
+  title?: string;
+  tracks?: Track[];
```

Comment:
I think the purpose of this nullables is to load sequence from external.
Why not just make a 'LoadingSequenceState'? Why should these be nullable and make a different type 'TimelineState' again?

Reply:
### `loadingSequenceState` can be removed or overwrited by `sequenceListView` and `topBar` or others.
Then it loses editing tracks.
So it is safe to make timeline own tracks.

### Why should tracks and title be nullable
Now, timeline editor is opened depending on the existence of title and tracks.

### Why different type 'TimelineState' again?
If existence of title and tracks were checked at top level. It would be convenient because we don't have to check it in editors